### PR TITLE
Incorrect errormessage when publishing with a domainId larger than 255 chars

### PR DESCRIPTION
**Pull request by vishnu-alapati**
### Applicable Issues

https://github.com/eiffel-community/eiffel-remrem-publish/issues/172

### Description of the Change

Provided proper error message when domainId is too long than expected.

### Benefits

It gives proper error message, when domainId is too long. so that user can have proper idea on the issue.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:  Vishnu Alapati vishnu.alapati@tcs.com

### CODE REVIEW

**Comment by elebroerik** [comment]
Seems fine from my side, Is it possible to test this someway?

File: publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java [hunk 1]
```diff
@@ -481,6 +481,9 @@ public void shutdownCompleted(ShutdownSignalException cause) {
         } catch (TimeoutException e) {
             log.error("Failed to publish message due to " + e.getMessage());
             throw new TimeoutException("Timeout waiting for ACK " + e.getMessage());
+        } catch (IllegalArgumentException e) {
+            log.error("Failed to publish message due to " + e.getMessage());
+            throw new IllegalArgumentException("Limit Exceeded " + e.getMessage());
```
Comment (Vaishnavi-Enugala) [inline publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java:486]: Error message should be more specific

Comment (vishnu-alapati) [reply publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java]: Updated the error response

**Review by Vaishnavi-Enugala — APPROVED** [review]

**Comment by vishnu-alapati** [comment]
> Seems fine from my side, Is it possible to test this someway?

yes, provide improper event which contains domainId length greater than 255 chars

**Review by jainadc9 — APPROVED** [review]

**Pull request merged**